locale/en.yml

```
@@ -13,17 +13,25 @@ flarum-suspend:
 
   # Translations in this namespace are used by the forum user interface.
   forum:
+    # These translations are used in the suspension info modal
+    infomodal:
+      dismiss_button: Dismiss
+      indefinite: This is an indefinite suspension
+      limited: "This suspension will be in force until {date}"
+      title: This account is suspended
     # These translations are used in the suspension notifications
```

Just matches the existing standard of blank lines between sections
```suggestion
      title: This account is suspended

    # These translations are used in the suspension notifications
```